apps/chat/src/components/Promptbar/components/PromptFolders.tsx

```diff
@@ -97,6 +97,12 @@ const PromptFolderTemplate = ({
     (state) => PromptsSelectors.selectChosenFolderIds(state, prompts),
   );
 
+  const emptyFoldersIds = useAppSelector(PromptsSelectors.selectEmptyFolderIds);
+
+  const isFolderEmpty = useAppSelector((state) =>
+    PromptsSelectors.selectIsFolderEmpty(state, folder.id),
+  );
```

Comment:
Why not `const isFolderEmpty = emptyFoldersIds.includes(folder.id);` (with `useMemo`)?

Reply:
I think it is better to have separate selector for this for the reusability.